### ISSUE TITLE
Upgrade SuiteCrm 7.02 to SuiteCRM7/1... Call popup does not work anymore

### ISSUE THREAD

**Issue by gipsalternative** [blak3r/yaai#185]
Hi,
After last upgrade Sugar, Callinize stopped running.
AMI is successfully connected to Callinize waiting for events.
asterisk_logger is pointing on the right place /var/www/gip..., user is set on www-data
So:
1/ dialout.js are present in front of all extensi are ons but when i click on... just moving but no connexion wit our iphone. Before, it was.
2/ Call popup is not displayed anymore so tranfers are, of course, not running to Zoiper.

But  i can see the inbound/outbound extensions and calls in Call module if i try to call from Zioper.. or someone call me.

The problem is i can't uninstall Callinize cause there is a base from 170 000 lead, contacts with phone numbers. If i do this, I will have troubles with "null" fields... 

I think the upgrade modified somethong for js files... Can u help me, please?

**Comment by blak3r** [comment]
What would be a null field? 

It's most likely some JavaScript error. I have no idea. Never have used suite crm. 

Sent via mobile

**Comment by gipsalternative** [comment]
Hi and thanks for your answer... 
SuiteCRM is Sugar CE equivalent to the Pro version. There  is not any difference. 
Of course you are right, it is a js problem.  So, is there a safe way to uninstall Callinize without loosing field?  

Cause in Studio last time i try to uninstall on another server, inbound, outbound and dialmagic where converted in null fields. Uninstalling does not erase the 3 fields created by Callinize module... For informations, i get base with 170 000 leads and phone fields already created....and i don't want to loose them! THKS

**Comment by ViktorNova** [comment]
For what it's worth, I upgraded from Sugar CE 6.something to Suite CRM 7 a few weeks ago, and everything started working again great, but only after uninstalling YAAI and then reinstalling it and re inputting all of the settings. Also this might have been a coincidence, but after upgrading, the call logger 'daemon' didn't seem to want to start at boot anymore and I had to start it by hand by doing 'php asterisklogger.php' (or something like that) and after that everything worked beautifully

Suite CRM is a fork of Sugar that basically takes Sugar CE (it's not a complete fork in the way that vTiger is, they stay current with upstream) and makes it much nicer out of the box, the whole experience feels much better and dare I say modern than stock Sugar CE
http://suitecrm.com/

**Comment by blak3r** [comment]
You should be able to uninstall. Do a repair and rebuild and then just remove any fields left behind in studio. 

Viktor, not sure what's causing the daemon issues. If you find our let me know. 

Blake Robertson
Callinize
blake@callinize.com

**Comment by ViktorNova** [comment]
Yes, I should have mentioned that in my process one or two repair & rebuilds were necessary, after uninstalling YAAI, and then again after reinstalling it

I will totally report back if I figure out the daemon issue, it is really strange. I am seriously considering installing Upstart on this Debian server _just_ to handle the asterisk logger, because it's way more straightforward than Sys V init, and I've noticed that it consistently will stop working at some point, and Upstart makes it really easy to respawn things if they crash

**Comment by blak3r** [comment]
Yeah I hate init scripts. 

Sent via mobile

**Comment by gipsalternative** [comment]
Hi! Solved :) I had to reinstall 3 times. SuiteCRM broke js files. Solution is: 
1/uninstall Callinize, Asterisk Logger,  restart Apache, got to Suite CRM, do a repair js files.. 
2 / Then, do a new install of Callinize, do a repair and rebuild... Set Users profiles under Studio (all parameters are stored, so nothing to do),  Reinstall and set Asterisk Logger, start it!!!!!!!!!

**Comment by blak3r** [comment]
Nice. Not sure why it's such a hassle. Everything in Callinize is 100% upgrade safe. 

Glad you figured it out. 

If you get tired of dealing with the logger and all that jazz. I recommend trying out our new product. www.callinize.com. 

Blake Robertson
Callinize
blake@callinize.com

**Comment by erestor-elensar** [comment]
I had the same problem, using the solution of @gipsalternative worked also for me

**Comment by blak3r** [comment]
Thanks @gipsalternative!

**Comment by gipsalternative** [comment]
You're welcome :)
